spec/ulid_spec.cr

```diff
@@ -56,5 +56,39 @@ describe ULID do
         (ulid_2 < ulid_1).should be_true
       end
     end
+
+    describe ".valid? : Bool" do
+      it "validate a valid string" do
+        ULID.valid?("01B3EAF48P97R8MP9WS6MHDTZ3").should eq true
+      end
+
+      it "should not validate invalid strings" do
```

Comment:
Recommend splitting this up into what is being tested. E.g. `it "detects incorrect length"`, `it "detects invalid characters"` etc